internal/config/main.go

```diff
@@ -12,7 +12,8 @@ import (
 )
 
 const (
-	defaultCollectorConfigMapEntry = "cwagentconfig.json"
+	defaultCollectorConfigMapEntry       = "cwagentconfig.json"
+	defaultTargetAllocatorConfigMapEntry = "targetallocator.yaml"
```

Comment:
What will be the contents of this file ? Is it the pod mapping to Prometheus targets or the prometheus scrape config YAML itself ?

Reply:
The latter.
```
targetallocator.yaml:
----
allocation_strategy: least-weighted
config:
  scrape_configs:
  - job_name: cloudwatch-agent
    scrape_interval: 10s
    static_configs:
    - targets:
      - 0.0.0.0:8888
label_selector:
  app.kubernetes.io/component: amazon-cloudwatch-agent
  app.kubernetes.io/instance: amazon-cloudwatch.cloudwatch-agent
  app.kubernetes.io/managed-by: amazon-cloudwatch-agent-operator
  app.kubernetes.io/part-of: amazon-cloudwatch-agent
prometheus_cr:
  scrape_interval: 30s
```